tests/NumberFormatterTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Tests;
+
+use App\Services\NumberFormatter;
+use PHPUnit\Framework\TestCase;
+
+class NumberFormatterTest extends TestCase
+{
+    public function testValidNumber()
```

Comment:
Tikėjausi, kad būtų panaudotas ir `@dataprovider` (liet. _parametriniai testai_)

Patikslinau skaidrėse:
![image](https://user-images.githubusercontent.com/1453957/58270250-fa06f280-7d91-11e9-8ca8-a8c5c1162e90.png)

Plačiau:
 * https://phpunit.readthedocs.io/en/8.1/writing-tests-for-phpunit.html#data-providers